app/queries/news_items/with_filters.rb

```diff
@@ -11,6 +11,15 @@ def self.call(
     )
       result = relation
 
+      # Apply FeedSources filter
+      if feed_sources.blank?
+        # Default feed sources
+        feed_sources = FeedSource.all
```

Comment:
Yikes, just realized this should be `FeedSource.active` instead of `FeedSource.all`...

Reply:
@anthotsang @jessehon I'm gonna make this change, then please lemme know if anything else looks amiss, I think this PR is ready to go after the fix.